### PR TITLE
Transaction endpoints

### PR DESCRIPTION
**Pull request by kohpai**

### CODE REVIEW

File: database/init.go [hunk 1]
```diff
@@ -93,8 +93,8 @@ func createConstantTable(tableName string) (err error) {
 	query :=
 		fmt.Sprintf("CREATE TABLE IF NOT EXISTS %s (", tableName) +
 			`name character varying(20) NOT NULL PRIMARY KEY,
-			created_at timestamp with time zone DEFAULT CURRENT_TIMESTAMP,
-			updated_at timestamp with time zone DEFAULT CURRENT_TIMESTAMP,
+			created_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
+			updated_at timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
 			deleted_at timestamp with time zone);`
```
Comment (oat9002) [inline database/init.go]: What is this for? If we delete, the record will be lost. Or there is other column to mark it as deleted

Comment (kohpai) [reply database/init.go]: If you use the endpoint `delete`, the `deleted_at` column will be set instead of literally delete the record.

**Comment by oat9002** [comment]
😂

**Comment by oat9002** [comment]
:1st_place_medal:

**Comment by kohpai** [comment]
@oat9002 We don't, but I want to, Lol

**Review by oat9002 — COMMENTED** [review]
:1st_place_medal:

**Comment by kohpai** [comment]
This is soooooo EPIC. It took me several months to finish (mostly because of my laziness tho). @oat9002 @filmsklz let's celebrate!!!

**Pull request merged**